### DISCUSSION TITLE
Guess the song 🎸🥁

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#826] [Galaxy Trivia] [answered]
```
G|-----|-------------------|----------------------|-------------------|----------------------|
D|-----|-------------------|----------------------|-------------------|----------------------|
A|-----|-------------------|----------------------|-------------------|----------------------|
E|-5-3-|-0---0---0-------0-|-0--0--3--0-5-----5-3-|-0---0---0-------0-|-0--0--3--0-5---------|
```

**Bonus:** Name the songwriter.

**Comment by peterbaccaro** [comment]
The song is **Another One Bites The Dust** by Queen.

**Comment by SANKARAMDAS** [comment, marked as answer]
"Another One Bites the Dust" is a song by the British rock band Queen. Written by bassist John Deacon, the song was featured on the group's eighth studio album The Game 1980.

**Reply by ParanoidUser** [reply]
Yahoo! 🎊🎉 Congrats on unlocking the cosmic-level 🥇 [Galaxy Brain](https://github.com/SANKARAMDAS?tab=achievements&achievement=galaxy-brain)! 👏👏 Welcome to the golden club, where fun reigns supreme. Keep rockin' it! 🚀

**Comment by hasanerzi** [comment]
"[Seven Nation Army](https://www.youtube.com/watch?v=0J2QdDbelmY)" by The White Stripes.
